### ISSUE TITLE
Minions Treasure Hunt

### ISSUE THREAD

**Issue by cherinae** [OpenSUTD/Discovery-Week-Working-Title-#11]
**Session Name**
Minions Treasure Hunt
_finding places, making friends, winning goodies!_

**Abstract**
The treasure hunt aims to encourage freshies to explore different parts of the school, familiarising themselves in the new environment while having fun.

**Student Organisation organising this session**
Minions

**Session intent (what to expect)**
Participants will get to know the different spaces in school while bonding with their new friends. They will also get a taste of SUTD's welfare events and are open to suggest welfare events that Minions can hold in the future.

**Session details**
When: 21 - 24 May (Tues - Fri)
Where: Key locations around the school
Participants: All freshmores only

Freshies have to form teams of 4 to participate. They will be looking for wooden laser cut minions. Each minion is tagged with a unique serial number in which they will key into a telegram bot. Prizes will be scheduled to be claimed on a separate day (28 May, 8pm tentative) and there is a limit of one prize per group. More information on the day and time will be disseminated via the bot.

**Logistical details**
We are looking at 40 prizes catering to 160 students. Types of prizes will be decided when budget is confirmed. We are looking at about $2/pax + miscellaneous costs, total budget ~$350

Emails: wangsui_yee@mymail.sutd.edu.sg, dominic_lim2@mymail.sutd.edu.sg

**Comment by Fishbiscuit** [comment]
Hello, it would greatly help if you could let us know some details of how the last session went. Specifically the actual turn out, feedback from students and the proceedings of the event please

**Comment by cherinae** [comment]
The last treasure hunt was held from 25th to 29th March 2019, where 119 eggs were hidden all across the campus, of which 77 were found. Participation rate was high, with most eggs found during the first day. Most of the eggs that not found were hidden in secluded locations. 

For this treasure hunt, the tokens will be hidden in popular key locations around campus. The motive is to not only let the freshies have fun, but also familiarise them with key locations in the campus. Difficulty level of this upcoming treasure hunt will thus be easier. Also, freshies will be relatively less busy compared to the participants of the previous treasure hunt. Hence, we expect most of the tokens to be found.

During the previous treasure hunt, we used $700 to purchase rewards and miscellaneous items such as decorations. For this upcoming event, we are toning down on the expenditure to approximately $2 per participant. As mentioned above, if most tokens were to be found, we require about $350 for the event.

From the feedback we gathered, treasure hunts are one of the more engaging events. Some participants even woke up at 8am to hunt for eggs. It was one of the more successful events that minions have organised.

**Comment by cherinae** [comment]
> Hello, it would greatly help if you could let us know some details of how the last session went. Specifically the actual turn out, feedback from students and the proceedings of the event please

The last treasure hunt was held from 25th to 29th March 2019, where 119 eggs were hidden all across the campus, of which 77 were found. Participation rate was high, with most eggs found during the first day. Most of the eggs that not found were hidden in secluded locations.
For this treasure hunt, the tokens will be hidden in popular key locations around campus. The motive is to not only let the freshies have fun, but also familiarise them with key locations in the campus. Difficulty level of this upcoming treasure hunt will thus be easier. Also, freshies will be relatively less busy compared to the participants of the previous treasure hunt. Hence, we expect most of the tokens to be found.
During the previous treasure hunt, we used $700 to purchase rewards and miscellaneous items such as decorations. For this upcoming event, we are toning down on the expenditure to approximately $2 per participant. As mentioned above, if most tokens were to be found, we require about $350 for the event.
From the feedback we gathered, treasure hunts are one of the more engaging events. Some participants even woke up at 8am to hunt for eggs. It was one of the more successful events that minions have organised.

**Comment by Fishbiscuit** [comment]
just to confirm, you don't need a specific venue yes?
also there doesn't seem to be a need for a formal sign up process as it's going to be done through telegram, could I clarify on this please!
Thank you :)

**Comment by cherinae** [comment]
> just to confirm, you don't need a specific venue yes?
> also there doesn't seem to be a need for a formal sign up process as it's going to be done through telegram, could I clarify on this please!
> Thank you :)

Yup, we do not need a specific venue. However, the prize collection will be held on 28 May at TT17 (we are booking it on our side). 
There is no formal sign up as well. They are only required to fill in their names and ID via a form given on telegram if they were to find a token for prize collection purposes.

**Comment by tlkh** [comment]
Please verify this and feel free to tell us if you want changes

https://discover.sutd.dev/events.html#treasure-hunt
